// config/liste.js
var crypto = require('crypto');
var rand = require('csprng');
var mongoose = require('mongoose');
var gravatar = require('gravatar');
var mod = require('config/models');
var MongoClient = require('mongodb').MongoClient, assert = require('assert');
var url = 'mongodb://mptdn:mptdn@ds027719.mlab.com:27719/parcautobdd';	
exports.listevoitures = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("voitures").find({});
		if(cursor!=null){
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
		}
		else callback("no result");
	});
}
exports.listevoituresdisponibles = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("voitures").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.listevoituresenmission = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("voitures").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.listeemployes = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("employes").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.listedemandes = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("affectations").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.listedemandesnonrepondus = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("affectations").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.listedepartements = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("departements").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.listetypeutilisateurs = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("typeutilisateurs").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.listetypeentretiens = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("typeentretiens").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.listetypevoitures = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("typevoitures").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.parametre = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("parametres").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
exports.lieu = function(callback) {
	MongoClient.connect(url, function(err, db) {
		var cursor = db.collection("lieus").find({});
		cursor.toArray(function(err, docs) {
			callback(docs);
			db.close();
		});
	});
}
