// config/liste.js
var crypto = require('crypto');
var rand = require('csprng');
var mongoose = require('mongoose');
var gravatar = require('gravatar');
var mod = require('config/models');
exports.listevoitures = function(callback) {
	mod.voiture.find({},function(err,result){
		callback(result);
	});
}
exports.listevoituresdisponibles = function(callback) {
	mod.voiture.find({},function(err,result){
		callback(result);
	});
}
exports.listevoituresenmission = function(callback) {
	mod.voiture.find({},function(err,result){
		callback(result);
	});
}
exports.listeemployes = function(callback) {
	mod.employe.find({},function(err,result){
		console.log(result.length);
		callback(result);
	});
}
exports.listedemandes = function(callback) {
	mod.affectation.find({},function(err,result){
		callback(result);
	});
}
exports.listedemandesnonrepondus = function(callback) {
	mod.affectation.find({},function(err,result){
		callback(result);
	});
}
exports.listedepartements = function(callback) {
	mod.departement.find({},function(err,result){
		console.log(result.length);
		callback(result);
	});
}
exports.listetypeutilisateurs = function(callback) {
	mod.typeutilisateur.find({},function(err,result){
		console.log(result.length);
		callback(result);
	});
}
exports.listetypeentretiens = function(callback) {
	mod.typeentretien.find({},function(err,result){
		console.log(result.length);
		callback(result);
	});
}
exports.listetypevoitures = function(callback) {
	mod.typevoiture.find({},function(err,result){
		console.log(result.length);
		callback(result);
	});
}
exports.parametre = function(callback) {
	mod.parametre.find({},function(err,result){
		console.log(result.length);
		callback(result);
	});
}
exports.lieu = function(callback) {
	mod.lieu.find({},function(err,result){
		console.log(result.length);
		callback(result);
	});
}
